### DISCUSSION TITLE
I like the anonaddy user interface

### DISCUSSION THREAD

**Discussion by Raytle** [j256/mailgw#12] [Ideas]
I like the [anonaddy](https://anonaddy.com/) user interface. Perhaps you might get some ideas by taking a look at it.

**Comment by j256** [comment]
Yeah that looks to be designed by a real frontend team.  I don't have that.  mailgw is my once and a while gig.  If you have specific recommendations with CSS, I'll probably incorporate it however.

**Comment by Raytle** [comment]
When you have a chance please see...

https://33mail.com/

and

https://news.ycombinator.com/item?id=25434753

You might be surprised how cheaply you can get stuff front end stuff done on Fiverr.com particularly if you are patient (willing to contact 50 or 100 freelancers). I would guess it could be done for $50 to $100. But almost all of the bids would be much higher. You wouldn't need every "seller" (that's what Fiverr calls freelancers) to be inexpensive; you'd only need one seller to be inexpensive. The seller would almost certainly be "new and hungry (eagerly looking for work).

In my experience, the key is go have extremely clear specifications and to find a female designer who lives in "Eastern Europe" (countries formerly behind what Churchill famously dubbed, "The iron curtain"). Specifically, I would recommend avoiding working with anyone in Asia east of Iran. I would recommend only working with a woman primarily because women's communication skills are generally vastly superior to those of men. Put simply: in my experience working with freelancers, men tend to ignore instructions and ignore criticism; whereas women tend to follow instructions and respond to criticism.

The easiest way to have **extremely** clear specifications would be to indicate something, "Make a copy this site using XYZ technologies." Then, after the front end for site had been completed, you could go in and change the fonts and colors yourself so that it looked a little bit different. Generally copying web designs is legal; that is, there's generally no copyright protection for web design. I suppose you could  probably get a decent logo designed on Fiverr for around $20. Something simple and boring would probably suffice. For a logos (and illustration/graphic design generally) I generally prefer working with women in Ukraine because their English skills are generally sufficient; they are often remarkably intelligent, and—I don't really know why (perhaps it's the schooling and/or culture)—Ukraine seems to have throngs of very talented designers and artists.
